### ISSUE TITLE
comment nav keyboard issues

### ISSUE THREAD

**Issue by andytuba** [honestbleeps/Reddit-Enhancement-Suite#1602]
honestbleeps>
when you hit "n" to toggle comment nav, 2 problems
[10:56pm] <honestbleeps>
1) it should be full opacity as if the mouse is over it
[10:56pm] <honestbleeps>
2) keyboard focus should be in the dropdown to select how to navigate
[10:56pm] <honestbleeps>
^^^ firefox fails #2 for some reason
